CONTRIBUTING.md

```diff
@@ -12,9 +12,11 @@ Note: This is the code development repository for *jQuery Core* only. Before ope
 
 ## Getting Involved
 
-We've put together [a short guide with tips, tricks, and ideas on getting started](http://contribute.jquery.org/open-source/). We're always looking for help identifying bugs, writing and reducing test cases, and documentation.
+[jQuery tenets](https://github.com/jquery/jquery/wiki/API-design-guidelines)
```

Comment:
Is `tenet` a common word? I haven't heard it before so it might a barrier to non-native speakers.